HousingSearchApi.Tests/ElasticSearchTests.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+using HousingSearchApi.V1.Infrastructure;
+using Nest;
+using NUnit.Framework;
+
+namespace HousingSearchApi.Tests
+{
+    [TestFixture]
+    public class ElasticsearchTests
+    {
+        protected ElasticClient ElasticsearchClient { get; private set; }
+
+        [OneTimeSetUp]
+        public void BeforeAllElasticsearchTests()
+        {
+            ElasticsearchClient = SetupElasticsearchConnection();
+        }
+
+        [SetUp]
+        public async Task SetupElasticsearchClient()
+        {
+            await BeforeAnyElasticsearchTest(ElasticsearchClient).ConfigureAwait(true);
+        }
+
+        [TearDown]
+        public void RunAfterAnyTests()
+        {
+            DeleteAddressesIndex(ElasticsearchClient);
+        }
+
+        public static async Task BeforeAnyElasticsearchTest(ElasticClient client)
+        {
+            DeleteAddressesIndex(client);
+
+            //TODO: Index?
+            await CreateIndex(Constants.EsIndex, client).ConfigureAwait(true);
+        }
+        public static ElasticClient SetupElasticsearchConnection()
+        {
+            var esDomainUri = Environment.GetEnvironmentVariable("ELASTICSEARCH_DOMAIN_URL")
+                              ?? "http://localhost:9202";
+            using var pool = new SingleNodeConnectionPool(new Uri(esDomainUri));
+            using var settings = new ConnectionSettings(pool).PrettyJson()
+                .DisableDirectStreaming()
+                .SniffOnStartup(false)
+                .ThrowExceptions();
+
+            return new ElasticClient(settings);
+        }
+
+        private static async Task CreateIndex(string name, IElasticClient client)
+        {
+            var settingsDoc = await File.ReadAllTextAsync("./../../../../data/elasticsearch/index.json")
```

Comment:
Could you not mark the index.json file that it should be copied to the output directory, then you wouldn't need such a huge relative path?

Reply:
That came in from the API I copied from :/ I'll look into this when I start working on ES